src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -19,29 +20,42 @@
 
     private final OfficerService officerService;
 
+    private Boolean officerValidationFlag;
+
     @Autowired
-    public CompanyOfficerValidation(OfficerService officerService){
+    public CompanyOfficerValidation(OfficerService officerService, Boolean officerValidationFlag){
         this.officerService = officerService;
+        this.officerValidationFlag = officerValidationFlag;
     }
 
     @Override
     public void validate(CompanyProfileApi companyProfileApi) throws EligibilityException, ServiceException {
         LOGGER.info("Validating Company Officers for: {}", companyProfileApi.getCompanyNumber());
+        if (!Boolean.TRUE.equals(officerValidationFlag)) {
```

Comment:
Once the feature flag is refactored to a primitive boolean. We can just do `!officerValidationFlag` which is significantly easier to read and understand